src/DGmethods/NumericalFluxes.jl

```diff
@@ -31,39 +31,47 @@ function diffusive_boundary_penalty! end
 struct CentralGradPenalty <: GradNumericalPenalty end
 
 function diffusive_penalty!(::CentralGradPenalty, bl::BalanceLaw,
-                            VF, nM, diffM, QM, aM, diffP, QP, aP, t)
-  DFloat = eltype(QM)
+                            σ, n⁻, G⁻, Q⁻, α⁻, G⁺, Q⁺, α⁺, t)
+  DFloat = eltype(Q⁻)
 
   @inbounds begin
-    ndim = 3
+    Nᵈ = 3
     ngradstate = num_gradient(bl,DFloat)
-    n_Δdiff = similar(VF, Size(ndim, ngradstate))
-    for j = 1:ngradstate, i = 1:ndim
-      n_Δdiff[i, j] = nM[i] * (diffP[j] - diffM[j]) / 2
+    ∇G = similar(σ, Size(Nᵈ, ngradstate))
```

Comment:
This is maybe a little confusing since it's not actually the gradient of `G`, but the normal time times `G`

Reply:
ah, when does the actual gradient of G happen then? 

I can change this line to be nG or n\hat G or or n\Delta G or some other variant

Reply:
After thinking about this some more, I think `∇G` makes sense. You are calculating the normal times the jump in `G`, which is the contribution to the DG gradient coming from the surface. the `diffusive!` function call then adds the contribution from the volume turning this variable into the full DG gradient of `G`.

does this interpretation make sense to you?

Reply:
@jkozdon bump

Reply:
I think that interpretation is mostly fine. What you are calling `∇G` isn't quite that though, but at the end of the day not sure your name is any worse than `n_Δdiff`. If it makes sense to you, I'm fine with it (it's defined right above, so someone can see what it is).

Reply:
I think that interpretation is mostly fine. What you are calling ∇G isn't quite that though, but at the end of the day not sure your name is any worse than n_Δdiff. If it makes sense to you, I'm fine with it (it's defined right above, so someone can see what it is).